modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-ingress-controller.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-ingress-control-config_{context}"]
+= Configuring ingress control in {microshift-short}
+
+You can use detailed ingress control settings by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+//Q: Any other prereqs?
+
+.Procedure
+
+. You can apply ingress control settings in either of the two following places:
+
+.. Update the {microshift-short} `config.yaml` configuration file by making a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory.
+
+.. Use a configuration snippet to apply the ingress control settings you want. To do this, create a configuration snippet YAML file and put it in the the `/etc/microshift/config.d/` configuration directory.
+
+[NOTE]
+====
+* After you create it, the `config.yaml` file takes precedence over built-in settings. The configuration file is read every time the {microshift-short} service starts.
+* Configuration snippet YAMLs take precedence over all settings anda `config.yaml` configuration file. See the Additional resources links for more information.
+====
+
+. When using the `config.yaml`, replace the default values in the `network` section of the {microshift-short} YAML with your valid values.
```

Comment:
Suggestion - if this is meant to be a conditional step and only applies to users who chose step 1a, I think it would be a little clearer to use "if" instead of "when": 
```suggestion
. If using the `config.yaml`, replace the default values in the `network` section of the {microshift-short} YAML with your valid values.
```

But I don't know the context of this procedure so i'll leave that to you

Reply:
Yeah, revised that, thanks. There are only two ways to configure MicroShift itself; either by using the config YAML, or by using config snippets. The snippets are new for 4.18, so still working out how to best include them as an option.